euth/comments/models.py

```diff
@@ -23,6 +25,12 @@ class Comment(TimeStampedModel):
     comment = models.TextField(max_length=1024)
     is_removed = models.BooleanField(default=False)
     is_censored = models.BooleanField(default=False)
+    ratings = GenericRelation(rating_models.Rating,
+                              related_query_name='comment',
+                              object_id_field='object_pk')
+    child_comments = GenericRelation('self',
+                                     related_query_name='parent_comment',
+                                     object_id_field='object_pk')
 
     class Meta:
         verbose_name = _("Comment")
```

Comment:
the sorting needs to be added here, as child comments are supposed to be sorted by created, the list of parent comments should be sorted by -created and you cant add the sorting to a nested serializer.